src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorageService.cs

```diff
@@ -126,20 +125,20 @@ protected override string GetDatabaseFilePath(string workingFolderPath)
                 }
                 throw;
             }
-
-            storage = sqlStorage;
-            return true;
         }
 
-        private static IDisposable TryGetDatabaseOwnership(string databaseFilePath)
+        private static IDisposable? TryGetDatabaseOwnership(string databaseFilePath)
```

Comment:
📝 This can be non-null again now

Reply:
I'll comment, but that's not the case. in the case of IOExceptions the IOUtilities helper will return null.